canonical-web-and-design/vanilla-framework
# Add name attribs to form inputs and buttons

## Done

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/867

the name attrib is for server side use - to denote what the value of the input is for as in a key-value pair. So I've limited the additions to only inputs / buttons / selects that look like they are intended for submission, or examples for use as submittable forms. Buttons used for frontend purpouses like toggling states in the ui do not need names as far as I understand. The name attrib itself is not mandatory.

## QA

- Pull code
- Run `./run`
- Test specifically existing search forms, verify they still work- Go through the list of changes, verify they are sensible and free of typos or ommissions
